.travis.yml

```diff
@@ -36,7 +36,15 @@ jobs:
         - LATEST_AZCOPY="$(curl -sLo- https://api.github.com/repos/Azure/azure-storage-azcopy/releases | jq -r '.[0].tag_name' | sed 's/^v//g')"
         - test -n "$LATEST_AZCOPY"
         - export LATEST_AZCOPY
+        - mkdir -p ~/.docker/cli-plugins
+        - wget -O - https://github.com/docker/buildx/releases/download/v0.5.1/buildx-v0.5.1.linux-amd64 > ~/.docker/cli-plugins/docker-buildx
+        - chmod a+x ~/.docker/cli-plugins/docker-buildx
+        - docker run --rm --privileged docker/binfmt:a7996909642ee92942dcd6cff44b9b95f08dad64
+        - docker buildx create --use --name mybuilder
       script:
-        - docker build --build-arg AZCOPY_VERSION="$LATEST_AZCOPY" -t docker-azcopy:$TRAVIS_COMMIT .
+
+        - docker login -u ${DOCKER_USERNAME} -p ${DOCKER_PASSWORD}
+        - travis_wait 30 docker buildx build --build-arg AZCOPY_VERSION="$LATEST_AZCOPY" -t $DOCKER_USERNAME/docker-azcopy:$TRAVIS_COMMIT --platform linux/amd64,linux/arm64 --push . >> /dev/null 2>&1
```

Comment:
What's the travis_wait here for?

Reply:
Firstly, I was facing max size exceeded error with huge amount of logs. So, I have redirected logs to /dev/null and after this build was running for more than 10 mins and as per the travis, build gets terminated if build is stalled at a command for more than 10 mins. Hence, I have increased waiting time to 30 mins using travis_wait.

Reply:
I wonder in this way if we can't know what's going on when something went wrong?